users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -719,41 +724,354 @@ def run_scf_baseline():
+        commit="c4d36d06f6465e82a50d400d114259e07b8b0709",
+    ).out_repository
+    returnn_root.hash_overwrite = "returnn_conv_padding"
+    report = run_nn_args(
+        nn_args,
+        report_args_collection,
+        dev_corpora,
+        returnn_root=returnn_root,
+        recog_args={"epochs": [350, 400, 450, "best"]},
+    )
+    return report
+
+
+def run_scf_specaug_sort():
+    gs.ALIAS_AND_OUTPUT_SUBDIR = "experiments/switchboard/ctc/feat/"
+
+    (
+        returnn_datasets,
+        rasr_loss_corpus_path,
+        rasr_loss_corpus_segments,
+        rasr_loss_lexicon_path,
+        dev_corpora,
+    ) = get_datasets()
+    returnn_args = {
+        "batch_size": 5000,
+        "rasr_binary_path": RASR_BINARY_PATH,
+        "rasr_loss_corpus_path": rasr_loss_corpus_path,
+        "rasr_loss_corpus_segments": rasr_loss_corpus_segments,
+        "rasr_loss_lexicon_path": rasr_loss_lexicon_path,
+        "datasets": returnn_datasets,
+        "extra_args": {
+            "accum_grad_multiple_step": 2,
+            "watch_memory": True,
+            "conv_pad_seq_len_to_power": 1.5,
+        },
+        "conformer_type": "wei",
+        "specaug_old": {"max_feature": 15},
+    }
+    feature_args = {"class": "ScfNetwork", "size_tf": 256 // 2, "stride_tf": 10 // 2}
+    lr_args = {
+        "peak_lr": 4e-4,
+        "start_lr": 1.325e-05,
+        "end_lr": 1e-5,
+        "increase_epochs": 180,
+        "decrease_epochs": 180,
+        "final_epochs": 0,
+    }
+
+    nn_args, report_args_collection = get_nn_args_baseline(
+        nn_base_args={
+            "bs2x5k_scf_specaugsortlayer2": dict(
+                returnn_args={**returnn_args, "specaug_old": {"max_feature": 15, "sort_layer2": True}},
+                feature_args=feature_args,
+                lr_args=lr_args,
+                report_args={"batch_size": "2x5k"},
+            ),
+            "bs2x5k_scf_specaugsortlayer2frome210": dict(
+                returnn_args={
+                    **returnn_args,
+                    "specaug_old": {"max_feature": 15, "sort_layer2": True},
+                    "extra_args": {
+                        "watch_memory": True,
+                        "conv_pad_seq_len_to_power": 1.5,
+                        "preload_from_files": {
+                            "existing-model": {
+                                "filename": (
+                                    "/work/asr4/vieting/setups/swb/work/20230406_feat/i6_core/returnn/training/"
+                                    "ReturnnTrainingJob.y9otnVMrBAWw/output/models/backup.epoch.210"
+                                ),
+                                "init_for_train": True,
+                            },
+                        },
+                    },
+                },
+                feature_args=feature_args,
+                lr_args={**lr_args, "peak_lr": 3.35e-4, "increase_epochs": 0, "decrease_epochs": 150},
+                report_args={"batch_size": "2x5k"},
```

Comment:
You copied this part from me apparently. I would prefer not to have it in this PR.